help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md

```diff
@@ -0,0 +1,109 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Create a SugarCRM Accounts & Contacts source connection in the UI 
+description: Learn how to create a SugarCRM Accounts & Contacts source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Create a [!DNL SugarCRM Accounts & Contacts] source connection in the UI
+
+This tutorial provides steps for creating a [!DNL SugarCRM Accounts & Contacts] source connector using the Platform user interface.
+
+## Overview
+
+[[!DNL SugarCRM]](https://www.sugarcrm.com/) is a customer relationship management (CRM) system. [!DNL SugarCRM]'s functionality includes sales-force automation, marketing campaigns, customer support, collaboration, Mobile CRM, Social CRM and reporting.
+
+To retrieve relationship data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the below [!DNL SugarCRM] APIs :
+   1. [Account API](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) to retrieve your companies, and the 
+   1. [Contact API](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) to retrieve the people. 
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Accounts & Contacts] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Account and Contact APIs.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```

Comment:
We can remove this - I will create a separate page to contain the pre-reqs for SugarCRM

Reply:
Removed from UI
  ```help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md```
Have preserved it within API
  ```help/sources/tutorials/api/create/crm/sugarcrm-events.md```